helpers/controllers/submissionController.ts

```diff
@@ -23,12 +23,14 @@ export const createSubmission = async (
   try {
     if (!args) throw new Error('Invalid args')
     const { challengeId, cliToken, diff, lessonId } = args
+    //to revert once CLI is fixed
+    const fixedDiff = diff.replace(/(.?\[\d*m)/g, '')
```

Comment:
if you do this here, why do it on the front end?

Reply:
This is for handling new submissions for the interim until CLi is fixed